### ISSUE TITLE
Total price column

### ISSUE THREAD

**Issue by evgenimelan** [partkeepr/PartKeepr#1254]
# Feature description

[comment]: # It would be useful to have a "total price" field. This "total price" should be calculated by Avg. Price * Stock.
(Average Price times Stock level)
This field could be then added as a column in the parts list view, so you have an overview over how much money is tied to a specific component. 
This is very useful for inventory reports.

**Comment by mhtvsSFrpHdE** [comment]
Yes, I'm looking for a method to get total price. It seems can only be done via stock history, however the export function is not work in my browser Firefox.

**Comment by mhtvsSFrpHdE** [comment]
I'm using v1.4.0 released at April 28, 2018, however the issue #359 shows it should be already fixed in Sep 30, 2015.  
This is confused.

I found that use browser developer tool, I can get HTML source code of `<table>` of stock history.  
It is possible to use python as a temporary solution...
